test/EndToEndTests/Tests/Client/Microsoft.OData.Client.E2E.Tests/Common/Server/Default/DefaultDataSource.cs

```diff
@@ -756,13 +762,15 @@ public void Initialize()
                         Amount = 1.9,
                         ExperationDate = new DateTimeOffset(new DateTime(2013, 12, 30))
                     },
-                    MyPaymentInstruments =
-                    [
+                    MyPaymentInstruments = new List<PaymentInstrument>()
+                    {

```

Comment:
question: is there a reason to switch from collection shorthand in tests? non-blocking

Reply:
No reason at all. You think we should use the short-hand?

Reply:
I've update this to use the shorthand